docs/source/routing/self-hosted/index.mdx

```diff
@@ -18,31 +18,31 @@ flowchart LR;
   class clients secondary;
 ```
 
-Each release of Apollo Router includes:
+For each version of the Apollo Router, Apollo provides:
 
-- A binary
-- A container image
-- A Helm chart
+- [A Helm chart for Kubernetes](#kubernetes-using-helm)
+- [A binary](#local-binary)
+- [A Docker image](#container)
 
-## Local binary
+## Kubernetes Using Helm
 
-Running Apollo Router directly from its binary speeds up local development, and it enables embedded use cases where containers are unavailable. 
+Helm is a package manager for Kubernetes. Apollo provides an application Helm chart with each release of Apollo Router in GitHub Container Registry. Since router v0.14.0, Apollo has released each router Helm chart as an Open Container Initiative (OCI) image in `oci://ghcr.io/apollographql/helm-charts/router`.
```

Comment:
```suggestion
Helm is a package manager for Kubernetes. Apollo provides a Helm chart with each release of Apollo Router in the GitHub Container Registry. Since router v0.14.0, Apollo has released each router Helm chart as an Open Container Initiative (OCI) image in `oci://ghcr.io/apollographql/helm-charts/router`.
```